CMakeLists.txt

```
@@ -4,6 +4,10 @@
 
 cmake_minimum_required(VERSION 3.5)
 PROJECT(GazeboYARPPlugins)
+option(USE_NEW_WRAPPERS "use new nws and nwc instead of old ones" OFF)
```

Using generic names it is dangerous as it can create clashes with other libraries. Could it make sense to prefix this with `GAZEBO_YARP_PLUGINS` as done with the other options defined in this project? 
```suggestion
option(GAZEBO_YARP_PLUGINS_USE_NEW_WRAPPERS "use new nws and nwc instead of old ones" OFF)
```


Furthermore, the name and description of the option feels a bit misleading to me (unless I am missing something). This option does not switch to use the new wrappers, but simply if enabled do not instantiate any network wrapper in the controlboard and sensor Gazebo plugins, leaving the task of opening the required network wrappers to the `gazebo_yarp_robotinterace` plugin. Ideally we should use a properly descriptive variable name, something like `GAZEBO_YARP_PLUGINS_DISABLE_IMPLICIT_NETWORK_WRAPPERS`  or similar, suggestions are welcome.

---

Yeah you're right, actually your description is correct. Both suggestion are good, I think that GAZEBO_YARP_PLUGINS_DISABLE_IMPLICIT_NETWORK_WRAPPERS can be good, i don't know if anyone has another suggestion, @randaz81 @drdanz

---

Let's proceed  with this, if anyone has a better idea we can always change it later.